### ISSUE TITLE
[finiteReco] Doc links to external pages (Trac #1218)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1212]
<http://web.physik.rwth-aachen.de/~icecubemgr/?n=LowEnergy.FirstGuessOfStartAndStopPoint>

A lot of good documentation currently lives in Aachen.  This is highly sub-optimal since these tend to stop working once a collaborator moves on and their account is deactivated.  Let's move these into our repo while we still can.  Maybe Aachen is set up to keep 'icecubemgr' alive indefinitely, but is it possible for anyone outside of Aachen to edit these docs?

Migrated from https://code.icecube.wisc.edu/ticket/1218
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:14:15", 
    "description": "<http://web.physik.rwth-aachen.de/~icecubemgr/?n=LowEnergy.FirstGuessOfStartAndStopPoint>\n\nA lot of good documentation currently lives in Aachen.  This is highly sub-optimal since these tend to stop working once a collaborator moves on and their account is deactivated.  Let's move these into our repo while we still can.  Maybe Aachen is set up to keep 'icecubemgr' alive indefinitely, but is it possible for anyone outside of Aachen to edit these docs?", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335655846260", 
    "component": "combo reconstruction", 
    "summary": "[finiteReco] Doc links to external pages", 
    "priority": "major", 
    "keywords": "", 
    "time": "2015-08-19T20:04:11", 
    "milestone": "", 
    "owner": "seuler", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150824T19:56:00: 
* seuler changed status from "new" to "accepted"
* seuler changed priority from "blocker" to "major"
* seuler commented:

> I don't think this is a blocker, since there will be an 'icecubemgr' account in Aachen as long as there is an IceCube group. That being said, it's true that the pages should be moved (and updated), but that is something I won't be able to do within a few days.

**Comment by icecube-trac** [comment]
Trac update at 20150911T17:47:42: 
* nega commented:

> > [1:41 PM] Sebastian Euler: Hi @olivas, I have only one ticket left (#1212), and that is about the documentation being on the Aachen wiki instead of the general !IceCube wiki. I don't think something like this should hold up meta-project releases. I would like to move the documentation at some point, so I don't want to close the ticket as wontfix. Can I leave it open and change the priority to minor, or something like that?


* nega changed milestone from "IceRec V5" to "December 2015"

**Comment by icecube-trac** [comment]
Trac update at 20151208T19:51:14: 
* olivas changed milestone from "December 2015" to "February 2016"
* olivas commented:

> Kicking this down the road again.  We should really try to get these ported to our repo, so that other's outside of Aachen could potentially edit them.

**Comment by icecube-trac** [comment]
Trac update at 20151208T21:55:28: 
* seuler changed status from "accepted" to "closed"
* seuler commented:

> Actually, I was going to close this later today, because https://wiki.icecube.wisc.edu/index.php/FiniteReco


* seuler changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20151209T14:44:21: 
* olivas commented:

> I hate to do this, but I need to reopen this ticket.  We're trying to get away from using wikis to document code and prefer rst or dox.  The reason is versioning.  What version of the code does that wiki correspond to?  The last release?  The trunk?  What of someone is using an old version of IceRec and wants to read the docs for the version of finiteReco they're actively using?  There's no question what version of the code the documentation refers to when it lives right beside the code in resources/docs.


* olivas changed resolution from "fixed" to ""
* olivas changed status from "closed" to "reopened"

**Comment by icecube-trac** [comment]
Trac update at 20151209T14:59:46: 
* seuler changed status from "reopened" to "accepted"
* seuler commented:

> Well, this doesn't come as a surprise, but it was worth a try... Obviously, the wiki contains much more information than the built-in documentation needs. So, is there a good model of what the built-in documentation should look like? Also, do we prefer rst or dox?

**Comment by icecube-trac** [comment]
Trac update at 20151209T15:13:07: olivas commented:

> We prefer rst for project level docs, but will take dox too.  Doxygen is generated anyway and is nice to point to from rst docs.  Check out the trigger-sim docs for an example : http://software.icecube.wisc.edu/simulation_trunk/projects/trigger-sim/index.html
> 
> I think most of this could be ported to rst, with the exception of "FiniteReco.StopPoint" which looks like it was removed a long time ago, according to that wiki.
> 
> My feeling is project-level docs (like what you have in the wiki) should be in rst and how the code works should be inline dox.

**Comment by icecube-trac** [comment]
Trac update at 20151215T13:12:59: 
* seuler commented:

> In changeset 140467:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="140467"
> updated release notes. Final commit of large documentation update. Closes #1212.
> ```


* seuler changed resolution from "" to "fixed"
* seuler changed status from "accepted" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:15: 
* olivas commented:

> Milestone February 2016 deleted


* olivas changed milestone from "February 2016" to ""
